pymathics/natlang/main.py

```diff
@@ -1139,36 +1158,36 @@ def _standard_property(
     ):
         senses = self._senses(py_word, wordnet, language_code)
         if not senses:
-            return Expression(SymbolMissing, "NotAvailable")
+            return Expression(SymbolMissing, StringNotAvailable)
         elif py_form == "List":
             word_property = WordProperty(self._short_syn_form, wordnet, language_code)
             property_getter = getattr(
                 word_property, "%s" % self._underscore(py_property), None
             )
             if property_getter:
-                return ListExpression(
+                return to_mathics_list(
```

Comment:
Is `to_mathics_list` preferred now?

Reply:
When not everything has been converted which is what happens here. I don't have the patience to figure how to get converted so that ListExpression can be used. 

When I wrote in the top-level comment: 

> Probably the code be revised so that to_expression, and to_mathics_list are not needed. That is deferred for another PR.